data/scenarios/Testing/1007-use-command.yaml

```diff
@@ -0,0 +1,83 @@
+version: 1
+name: The Use command
+creative: false
+description: Open a gate
+objectives:
+  - goal:
+      - Grab the flower on the other side of the gate.
+    condition: |
+      as base {has "flower"}
+solution: |
+  move;
+  k <- grab;
+  equip k;
+  use k forward;
+  move;
+  move;
+  grab;
+robots:
+  - name: base
+    dir: [1,0]
+    display:
+      char: Ω
+      attr: robot
+    devices:
+      - ADT calculator
+      - grabber
+      - logger
+      - treads
+      - welder
+entities:
+- name: fence
+  display:
+    attr: wood
+    char: '#'
+  description:
+  - Impassable barrier
+  properties: [known, unwalkable]
+- name: gate key
+  display:
+    attr: iron
+    char: 'k'
+  description:
+  - Can open a closed gate
+  capabilities: [drill]
```

Comment:
It's a bit strange the the capability is still named `drill`.  But maybe we'll generalize that in a future PR?

Reply:
In fact, no capability is required; I'll remove this line.